server/command.go

```diff
@@ -324,10 +341,39 @@ func (p *Plugin) runSettingsCommand(args []string, extra *model.CommandArgs) (bo
 		}
 
 		p.postCommandResponse(extra, responseMessage)
-	} else {
+
+	case "block_incoming":
+		if len(args) < 2 {
```

Comment:
Maybe this can show the current setting?

Reply:
Thank you for the feedback! I will apply the changes.